ClubMembersSortedCF/src/members/v1/ZipCodeValidator.java

```
@@ -0,0 +1,7 @@
+package members.v1;
+
+public class ZipCodeValidator {
+    boolean isValidZipCode(String zipCode){
+        return zipCode.matches("[1-9]\\d{3}");
```

smart validator

---

of course, you could implement a generic `Validator<T>` interface here where (with obviously replacing `T` with integer). not an issue, just +1 more opportunity to practice basic OOP

---

+1 thing: would be good to add some tests - for example for this validator or whenever we are expected to perform some logic. examples:
- you can test your builder
- you can test the case whenever you need to throw an exception (if the zip code is invalid)

In general, if you write something, it is your responsibility to ensure it works properly